src/Functions/reinterpretAs.cpp

```diff
@@ -75,9 +75,9 @@ class FunctionReinterpret : public IFunction
 
         if (result_reinterpret_type.isFixedString())
         {
-            if (!from_type->isValueUnambiguouslyRepresentedInFixedSizeContiguousMemoryRegion())
-                throw Exception("Cannot reinterpret " + from_type->getName() +
-                    " as FixedString because it is not fixed size and contiguous in memory",
+            if (!from_type->isValueUnambiguouslyRepresentedInFixedSizeContiguousMemoryRegion() && !WhichDataType{from_type}.isUUID())
```

Comment:
Update isValueUnambiguouslyRepresentedInFixedSizeContiguousMemoryRegion for UUID to return true.

Reply:
Ok.